notebooks/00_quick_start/sar_single_node_movielens.ipynb

```diff
@@ -41,7 +41,7 @@
   },
```

Comment:
<p>mmm can you check this? if we are using the test set, shouldn't we have users close to 600 (like in the previous commit= instead of 1?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>